CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* fix: Enable channel context menu for all users
```

Comment:
Am I using the changelog correctly here?

Reply:
Yes